gnuradio-runtime/lib/pmt/pmt_unv.cc

```diff
@@ -137,7 +138,7 @@ uint8_t* u8vector_writable_elements(pmt_t vector, size_t& len)
 
 const std::string pmt_u8vector::string_ref(size_t k) const
 {
-    return boost::lexical_cast<std::string, uint8_t>(ref(k));
+    return std::to_string(ref(k));
```

Comment:
I like this!

Reply:
Except it was actually a mistake. :-)

`lexical_cast` turns 0x42 into `B`, and `to_string` makes it `66`. Fixed.